core/src/test/java/org/apache/iceberg/view/TestViewMetadataParser.java

```diff
@@ -101,16 +101,20 @@ public void readAndWriteValidViewMetadata() throws Exception {
 
     String json = readViewMetadataInputFile("org/apache/iceberg/view/ValidViewMetadata.json");
     ViewMetadata expectedViewMetadata =
-        ViewMetadata.builder()
-            .assignUUID("fa6506c3-7681-40c8-86dc-e36561f83385")
-            .addSchema(TEST_SCHEMA)
-            .addVersion(version1)
-            .addVersion(version2)
-            .setLocation("s3://bucket/test/location")
-            .setProperties(
-                ImmutableMap.of("some-key", "some-value", ViewProperties.COMMENT, "some-comment"))
+        ViewMetadata.buildFrom(
```

Comment:
this (and the other places) is just wrapping it in a `ViewMetadata.buildFrom(..).setCurrentVersionId(2).build()` call, because `ValidViewMetadata.json` expects 2 versions in the `version-log`